packages/react-native-reanimated/src/Colors.ts

```diff
@@ -675,6 +686,24 @@ export function processColorsInProps(props: StyleProps) {
   for (const key in props) {
     if (ColorProperties.includes(key)) {
       props[key] = processColor(props[key]);
+    } else if (
+      NestedColorProperties[key as keyof typeof NestedColorProperties]
+    ) {
+      const nestedPropGroup = props[key] as StyleProps;
+      // most of the time there is only one nested prop in boxShadow array
```

Comment:
What's the purpose of this comment?

Reply:
as if someone were scared of nested loop 😳 remove it?

Reply:
Yeah, I think it's a bit misleading in this form.